Tella/Scenes/CommonConnectionReport/ViewModels/ReportMainViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ReportMainViewModel.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 1/7/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class ReportMainViewModel: ObservableObject {
+    
+    @Published var draftReportsViewModel : [CommonCardViewModel] = []
+    @Published var outboxedReportsViewModel : [CommonCardViewModel] = []
+    @Published var submittedReportsViewModel : [CommonCardViewModel] = []
+    
+    @Published var selectedCell: Pages = .draft
+    
```

Comment:
Maybe changig the name to `selectedPage` is more meaningful than `selectedCell` (it looks like UIKit) and changing the type `Pages` to `ConnectionPage` as it is one page and not an array of pages